### PR TITLE
sample size

### PR DESCRIPTION
**Pull request by wycalan**
I added a example that shows how to use simulations to estimate the sample size for a cluster-based permutation test. This is related to [issue #364](https://github.com/fieldtrip/website/issues/364)

### CODE REVIEW

**Comment by robertoostenveld** [comment]
Thanks, this is a very nice example - starting from a simple coin-tossing example and building up in complexity. I made some suggestions to the text (mainly formatting and a bit of reformulation in https://github.com/wycalan/v_samplesize/pull/1.

One small change that I made to the text but important to mention is that we should not speak of "significant clusters", as it is a non-parametric test where H0 states that the data is exchangeable, but it does not say anything about clusters. When we reject H0, we do that on the basis of the clusters. But the clusters (or the mean, or another parameter) are not what the inference is about: that is about the exchangeability.  See also https://www.fieldtriptoolbox.org/faq/how_not_to_interpret_results_from_a_cluster-based_permutation_test/

File: example/sample_size.md [hunk 1]
```diff
@@ -181,10 +181,10 @@ cfg.freq           = exampleData.freq;
 cfg.label          = exampleData.label;
 
 % parameters for the simulated cluster
-cfg.clusterfreq    = [4 8];                    % freq range you want to be included in the significant cluster,
-cfg.clustertime    = [0.03 0.25];              % time range you want to be included in the significant cluster,
-cfg.clusterchan    = {'CZ','C1','CPZ','CP1'};  % channles you want to be included in the significant cluster
-% These three parameters above should be set to be similar to those of the cluster of your preliminary results
+cfg.clusterfreq    = [4 8];                    % freq range you want to be included in the cluster displaying effect of interest,
```
Comment (robertoostenveld) [inline example/sample_size.md:184]: good catch, thanks!

**Comment by robertoostenveld** [comment]
Thanks for the merge and updates. Is it now ready to be published on the FieldTrip website? If so, I will merge. 

I also wanted to make a news item about this on the front page to make people aware of this, but that can be done after the weekend.

**Comment by wycalan** [comment]
> Thanks for the merge and updates. Is it now ready to be published on the FieldTrip website? If so, I will merge.
> 
> I also wanted to make a news item about this on the front page to make people aware of this, but that can be done after the weekend.
Hi, Robert. Thanks for your editing. The text is now definitely much smoother after your polishing the language. I agree that it's ready to be published on Fieldtrip website, and I love it that this example is going to be a news on the front page.

There is a small thing I want to discuss with you. I was wondering if it's OK that one sentence could be added at the end of the page acknowledging my contribution to this example. The sentence can go like "This example is contributed by [Dr. Cheng Wang](https://www.researchgate.net/profile/Cheng-Wang-93), and edited by Dr. Robert Oostenveld ". I don't know if it’s against Fieldtrip's policy or convention. If you don't like this idea, I am totally OK. You can still publish this example on Fieldtrip website without adding this sentence.

**Comment by wycalan** [comment]
> Thanks for the merge and updates. Is it now ready to be published on the FieldTrip website? If so, I will merge.
> 
> I also wanted to make a news item about this on the front page to make people aware of this, but that can be done after the weekend.

Hi, Robert. Thanks for your editing. The text is now definitely much smoother after your polishing the language. I agree that it's ready to be published on Fieldtrip website, and I love it that this example is going to be a news on the front page.

There is a small thing I want to discuss with you. I was wondering if it's OK that one sentence could be added at the end of the page acknowledging my contribution to this example. The sentence can go like "This example is contributed by [Dr. Cheng Wang](https://www.researchgate.net/profile/Cheng-Wang-93), and edited by Dr. Robert Oostenveld ". I don't know if it’s against Fieldtrip's policy or convention. If you don't like this idea, I am totally OK. You can still publish this example on Fieldtrip website without adding this sentence.

**Comment by robertoostenveld** [comment]
> There is a small thing I want to discuss with you. ... "This example is contributed by Dr. Cheng Wang" 

Yes, that is fair! The website is maintained by [many people](https://github.com/fieldtrip/website/people) and even more prior to moving it over to github a few years back. Your name will also show on github after merging and remembered for eternity, but that is not so visible for regular website visitors. 

In the future, this specific page might also be edited/maintained by others, so I will not add my own name. But as you are the original author and did 99.9% of the work, it is not more than fair that your name is highlighted. Giving more explicit credit to external contributors would actually also be good for some of the other pages on the website, as we really appreciate contributions like these!

**Comment by wycalan** [comment]
> > There is a small thing I want to discuss with you. ... "This example is contributed by Dr. Cheng Wang"
> 
> Yes, that is fair! The website is maintained by [many people](https://github.com/fieldtrip/website/people) and even more prior to moving it over to github a few years back. Your name will also show on github after merging and remembered for eternity, but that is not so visible for regular website visitors.
> 
> In the future, this specific page might also be edited/maintained by others, so I will not add my own name. But as you are the original author and did 99.9% of the work, it is not more than fair that your name is highlighted. Giving more explicit credit to external contributors would actually also be good for some of the other pages on the website, as we really appreciate contributions like these!

Many thanks. It's been a pleasure to be able to make a small contribution to your great toolbox. Fieldtrip is very helpful for my search, and I am teaching students in our department to use it.

**Pull request merged**